packages/hash/integration/src/tests/model/knowledge/link.model.test.ts

```diff
@@ -35,6 +35,25 @@ describe("Link model class", () => {
   let targetEntityFriend: EntityModel;
   let targetEntityAcquaintance: EntityModel;
 
+  const createEntityType = (params: { title: string }) =>
```

Comment:
If you want to introduce this, can you perhaps use it for places in the file where we're already doing entity type initialization?

Reply:
Sure, done in https://github.com/hashintel/hash/pull/1081/commits/567b33c226e7d2a4979e61f3d4c1066d3ea4746c